tutorials/shaders/advanced_postprocessing.rst

```diff
@@ -177,7 +182,7 @@ Now, attach a script to the MeshInstance and use the following code:
     mesh.add_surface_from_arrays(Mesh.PRIMITIVE_TRIANGLES, mesh_array)
 
 .. note:: The triangle is specified in normalized device coordinates. Recall, NDC run
-          from ``-1`` to ``1`` in both the ``x`` and ``y`` directions. This makes the screen
+          from ``0.0`` to ``1.0`` in both the ``x`` and ``y`` directions. This makes the screen
```

Comment:
```suggestion
          from ``-1.0`` to ``1.0`` in both the ``x`` and ``y`` directions. This makes the screen
```

Reply:
Is it actually between -1.0 and 1.0? I thought that was in OpenGL.